source/core/sudoku/sudoku.d

```diff
@@ -165,15 +165,15 @@ version(unittest)
 								[3, 1, 2, 6, 4, 5],
 								[6, 4, 5, 1, 3, 2]];
 
-	int[][] classic9x9 =  [[0, 0, 3, 2, 6, 0, 0, 0, 0],
-								[0, 0, 7, 0, 0, 1, 0, 2, 3],
-								[0, 8, 6, 0, 0, 0, 4, 0, 0],
-								[5, 0, 0, 0, 0, 8, 0, 9, 0],
-								[6, 4, 0, 3, 0, 7, 0, 1, 0],
-								[0, 0, 0, 0, 0, 0, 0, 0, 5],
-								[9, 2, 0, 0, 4, 0, 0, 0, 7],
-								[0, 0, 0, 0, 0, 5, 9, 8, 0],
-								[0, 0, 1, 6, 0, 0, 0, 3, 0]];
+	int[][] classic9x9 =   [[0, 0, 3, 2, 6, 0, 0, 0, 0],
```

Comment:
Extra space?

Reply:
Yeah, github is a bit weird with this, It has an extra space to align with the rest of the matrix. In text editor it is aligned correctly, however github has it's tabs to 8 spaces by default and there is no way to change it unless you edit the file directly in github and then commit the changes.